tests/app/status/test_status.py

```diff
@@ -0,0 +1,66 @@
+import json
+from ..helpers import BaseApplicationTest
+from requests import Response
+
+import mock
+from nose.tools import assert_equal, assert_in
+
+
+class TestStatus(BaseApplicationTest):
+    def setup(self):
+        super(TestStatus, self).setup()
+
+        self._api_response = mock.patch(
```

Comment:
These tests work fine as they are and I'm not saying we should re-do them or anything, but it's worth taking a look at the requests_mock package used here: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/tests/app/flask_api_client/test_api_client.py
I think it's a bit easier and clearer than building up the Response objects as you have here - worth bearing in mind for the future. (Or have a play with it here it the urge takes you).